templates/drupal-10/docker-compose.yml

```diff
@@ -79,6 +79,24 @@ services:
       # - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.middlewares=redirect-to-https"
       # - "traefik.http.middlewares.redirect-to-https.redirectscheme.scheme=https"
 
+  markdownlint:
+    image: itkdev/markdownlint
+    profiles:
+      - dev
+    volumes:
+      - ./:/md
+
+  prettier:
+    # Prettier does not (yet, fcf.
+    # https://github.com/prettier/prettier/issues/15206) have an official
+    # docker image.
+    # https://hub.docker.com/r/jauderho/prettier is good candidate (cf. https://hub.docker.com/search?q=prettier&sort=updated_at&order=desc)
+    image: jauderho/prettier
+    profiles:
+      - dev
+    volumes:
+      - ./:/work
+
```

Comment:
It may be a good idea to move this to the end of the file, i.e. after “the real” project services, to make it clear that these services are “only” used for code checks, e.g.

``` yaml
  …

  memcached:
    …

  # Code checks tools
  markdownlint:
    …

  …
```